doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -0,0 +1,97 @@
++------------+-----------------------------------------+-----------------+---------------------------+-------------------------------+---------------------------+---------------------------+-------------------------------+------------------+------------------+
+| End Device | :ref:`CoAP Client <coap_client_sample>` | Yes             |                       399 |                           145 |                        32 |                        80 |                            23 |              576 |              103 |
++------------+-----------------------------------------+-----------------+---------------------------+-------------------------------+---------------------------+---------------------------+-------------------------------+------------------+------------------+
+
+.. _thread_ot_memory_52833:
+
+nRF52833 DK RAM and flash memory requirements
+*********************************************
+
+The following table presents memory requirements for samples running on the :ref:`nRF52833 DK <gs_programming_board_names>` (:ref:`nrf52833dk_nrf52833 <zephyr:nrf52833dk_nrf52833>`) with the software cryptography support provided by the :ref:`nrfxlib:nrf_oberon_readme` module.
+
++------------+-----------------------------------------+-----------------+---------------------------+-------------------------------+---------------------------+---------------------------+-------------------------------+-------------------+------------------+
+| Role       | Sample                                  | Multiprotocol   |   ROM OT stack + app [kB] |   ROM Bluetooth LE stack [kB] |   Persistent storage [kB] |   RAM OT stack + app [kB] |   RAM Bluetooth LE stack [kB] | Total ROM [kB]    |   Total RAM [kB] |
++============+=========================================+=================+===========================+===============================+===========================+===========================+===============================+===================+==================+
+| Router     | :ref:`CLI <ot_cli_sample>`              | No              |                       455 |                             0 |                        32 |                        97 |                             0 | 487               |               97 |
++------------+-----------------------------------------+-----------------+---------------------------+-------------------------------+---------------------------+---------------------------+-------------------------------+-------------------+------------------+
+| Router     | :ref:`CLI <ot_cli_sample>`              | Yes             |                       583 |                           128 |                        32 |                        97 |                            17 | 743 (exceeds ROM) |              114 |
```

Comment:
What about two new columns showing the available flash and RAM? Then the `(exceeds ROM)` comment can be removed.